sdk/core/azure-core-amqp/pom.xml

```diff
@@ -80,6 +80,7 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
+      <version>${mockito-core.version}</version>
```

Comment:
I thought thim the parent pom.xmls would be inherited from

Reply:
Removed